ui/imports/utils/Constants.qml

```diff
@@ -613,6 +613,7 @@ QtObject {
     readonly property int communityChatInvitationOnlyAccess: 2
     readonly property int communityChatOnRequestAccess: 3
 
+    readonly property int activityCenterNotificationTypeNoType: 0
```

Comment:
This could be a QML `enum`, to mimic the `ActivityCenterNotificationType` from NIM, just saying :)

Reply:
> This could be a QML `enum`, to mimic the `ActivityCenterNotificationType` from NIM, just saying :)

Wow, thats Qt 5.15 feature? Do we have somewhere it used?

Reply:
Qt 5.14 imo but yeah, now we can rely on Qt 5.15 too

Reply:
We've had an enum there already in Constants.qml:
```qml
    enum ErrorType {
        SendAmountExceedsBalance,
        NoRoute,
        NoError
    }
```